packages/@expo/package-manager/src/node/NpmPackageManager.ts

```diff
@@ -82,6 +83,11 @@ export class NpmPackageManager extends BasePackageManager {
     return this.runAsync(['uninstall', '--global', ...namesOrFlags]);
   }
 
+  runBinAsync(command: string[], options: SpawnOptions = {}) {
+    this.log?.(`> npx ${command.join(' ')}`);
+    return spawnAsync('npx', command, { ...this.options, ...options });
```

Comment:
Doesn't this need to be split up with `--`? E.g. `npx eslint -- --max-warnings 0`?

Reply:
seems to work fine without `--`

```
npx eslint . --help
eslint [options] file.js [file.js] [dir]

Basic configuration:
...etc
```